plugins/modules/docker_container.py

```diff
@@ -1049,25 +1017,25 @@
     state: started
     restart: true
     links:
-     - "myredis:aliasedredis"
+      - "myredis:aliasedredis"
     devices:
-     - "/dev/sda:/dev/xvda:rwm"
+      - "/dev/sda:/dev/xvda:rwm"
     ports:
-     # Publish container port 9000 as host port 8080
-     - "8080:9000"
-     # Publish container UDP port 9001 as host port 8081 on interface 127.0.0.1
-     - "127.0.0.1:8081:9001/udp"
-     # Publish container port 9002 as a random host port
-     - "9002"
-     # Publish container port 9003 as a free host port in range 8000-8100
-     # (the host port will be selected by the Docker daemon)
-     - "8000-8100:9003"
-     # Publish container ports 9010-9020 to host ports 7000-7010
-     - "7000-7010:9010-9020"
+   # Publish container port 9000 as host port 8080
+      - "8080:9000"
+   # Publish container UDP port 9001 as host port 8081 on interface 127.0.0.1
+      - "127.0.0.1:8081:9001/udp"
+   # Publish container port 9002 as a random host port
+      - "9002"
+   # Publish container port 9003 as a free host port in range 8000-8100
+   # (the host port will be selected by the Docker daemon)
+      - "8000-8100:9003"
+   # Publish container ports 9010-9020 to host ports 7000-7010
```

Comment:
@russoz unindents also happen for comments in lists:
```suggestion
     # Publish container port 9000 as host port 8080
      - "8080:9000"
     # Publish container UDP port 9001 as host port 8081 on interface 127.0.0.1
      - "127.0.0.1:8081:9001/udp"
     # Publish container port 9002 as a random host port
      - "9002"
     # Publish container port 9003 as a free host port in range 8000-8100
     # (the host port will be selected by the Docker daemon)
      - "8000-8100:9003"
     # Publish container ports 9010-9020 to host ports 7000-7010
```